css/properties/place-self.json

```diff
@@ -32,10 +32,10 @@
                 "version_added": "43"
               },
               "safari": {
-                "version_added": false
+                "version_added": "11"
```

Comment:
Using http://mdn-bcd-collector.appspot.com/tests/css/properties/place-self I see it's not supported in Safari 10.1 and is supported in 11.1, so this seems right 👍